pyanaconda/modules/users/users_interface.py

```diff
@@ -107,6 +108,18 @@ def IsRootAccountLocked(self) -> Bool:
         """
         return self.implementation.root_account_locked
 
+    @emits_properties_changed
+    def SetRootPasswordSSHLoginAllowed(self, root_password_ssh_login_allowed: Bool):
```

Comment:
Please, add a unit test for the new DBus property.

Reply:
Added.